main.go

```diff
@@ -174,11 +185,21 @@ func runRecordings(conf *Config, camera *lepton3.Lepton3) error {
 
 		// If motion detected, allow minFrames more frames.
 		if motion.Detect(frame) {
-			if window.Active() {
+			writeLog := motionLogFrame <= totalFrames-(10*framesHz)
```

Comment:
the `writeLog` name is a bit generic/unclear. How about `shouldLogMotion` or something?